api/src/controllers/UserOrganisationSettingsController.js

```diff
@@ -43,7 +43,7 @@ const validateUpdatableKeys = (body, scopes) => {
 
   const updatingKeys = Object.keys(body);
 
-  if (scopes.includes(MANAGE_ALL_USERS)) {
+  if (scopes.some(s => [MANAGE_ALL_USERS, ALL].includes(s))) {
```

Comment:
- Allow ALL scope to update 'filter', 'roles', and 'scopes' in the user organisation settings.